### ISSUE TITLE
ID conflict

### ISSUE THREAD

**Issue by Puke84** [oitsjustjose/Miscellany#1]
I want to use/try your mod but i keep getting a ID conflict :30. Entity
The mod that it's conflicting with is Vampirism. i dont know whos end this is on or how it can be fixed.
just thought i would mention it.

**Comment by oitsjustjose** [comment]
Looking into a fix now, I might've messed up something somewhere with the EntityRegistry. If I can't fix it with what I'm working on now, then I'll make the EntityID configurable so that you can fix this conflict.

**Comment by oitsjustjose** [comment]
Have you got a link to the Vampirism mod? I'm seeing a few, but they're all WIP and for 1.7.10, so I'm not sure which one specifically.

**Comment by Puke84** [comment]
https://github.com/TeamLapen/Vampirism/releases

**Comment by oitsjustjose** [comment]
Is that the only other mod you have installed with Miscellany? Running the Miscellany 1.2 release from yesterday and Vampirism 0.4.3, I do not get a crash. And I was pretty sure Entity ID's were supposed to be mod-independent, so that it doesn't matter if my ID is 30 and his is too, because they're two different mods. I definitely could be wrong though, and like I said, I have a fix for it regardless of the situation, I just want to replicate the crash myself to see if / when I have fixed it.

**Comment by copygirl** [comment]
Full crash log with stack trace would be useful.

**Comment by Puke84** [comment]
thats the crash log. 

http://pastebin.com/Q8Td3Fts

**Comment by oitsjustjose** [comment]
Nope, it's "technically" on my end. That's not a conflict with the EntityID, but with the ID I'm using for adding spawn eggs for my entities. @copygirl  any thoughts on a fix? Since it's just metadata, should I just try to set my ID's to something a little higher and more random? I am currently just using 30 - 32

**Comment by copygirl** [comment]
Since the item metadata/damage can be pretty high, I'd just go for a random large value like 13206+ for the spawn egg ID, if that's possible (I think it is).
